widgets/use-assets/src/runtime/widget.tsx

```diff
@@ -0,0 +1,54 @@
+/** @jsx jsx */
+import {React, AllWidgetProps,  jsx } from 'jimu-core'
```

Comment:
```suggestion
import {AllWidgetProps,  jsx } from 'jimu-core'
```

Removing `React` from our `jimu-core` import as we're using the `AllWidgetProps` type which extends from `React`.

Reply:
I would agree here to remove the `React` import - is there a specific reason it's included here @qlqllu?

Reply:
Just checked, we can remove the `React` because of this directive: `/** @jsx jsx */`

Reply:
Thanks for clarifying @qlqllu! My understanding was incorrect.